src/sentry/api/urls.py

```diff
@@ -1085,6 +1086,11 @@
         r"^projects/",
         include(
             [
+                url(
+                    r"rule-conditions/$",
+                    ProjectAgnosticRuleConditionsEndpoint.as_view(),
+                    name="sentry-api-0-project" "-agnostic-rule" "-conditions",
```

Comment:
Why is this 3 strings?

Reply:
Cleaned up. It should have been one, thanks.